<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PoolMonitor</class>
 <widget class="QDialog" name="PoolMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>股池监测</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblNewStockCode">
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <property name="text">
        <string>股票代码/名称</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="iptStockCode">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
        </font>
       </property>
       <property name="text">
        <string>贵州茅台</string>
       </property>
       <property name="maxLength">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddStock">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>添加股票</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveStock">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>删除股票</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearStocks">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>清空列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnImportPool">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>导入股池</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tblStockList">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>80</number>
     </attribute>
     <column>
      <property name="text">
       <string>股票代码</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>股票名称</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>最新价格</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>3日涨跌幅</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>历史最高</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>回撤幅度</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>距离MA5</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>距离MA20</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>距离MA60</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>距离MA120</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>距离MA250</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
